### PR TITLE
wget-parser: init at 2.0.0

### PR DESCRIPTION
**Pull request by erdnaxe**
###### Description of changes

wget-parser is a tool to format as JSON the output of a wget spider scan. It is useful to scan a website for dead links.

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Kranzes** [comment]
Result of `nixpkgs-review pr 165945` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>wget-parser</li>
  </ul>
</details>

File: pkgs/tools/networking/wget-parser/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-GSCVaW57qnXTppttX5yItLjlIS9LKj5PwXSW1j0v+7Y==";
+  };
+  packageJSON = ./package.json;
+  yarnLock = ./yarn.lock;
+  yarnNix = ./yarn.nix;
+
+  postInstall = ''
+    ln -s $out/libexec/${pname}/deps/${pname}/bin/* $out/bin/
```
Comment (SuperSandro2000) [inline pkgs/tools/networking/wget-parser/default.nix:20]:
```suggestion
    ln -s $out/libexec/wget-parser/deps/wget-parser/bin/* $out/bin/
```

**Review by SuperSandro2000 — COMMENTED** [review]
Maybe we should put this into nodePackages?

**Comment by DamienCassou** [comment]
I'm closing this issue as there hasn't been any development for some time. Feel free to reopen with answers to reviewers.

**Pull request closed without merging**